src/Command/VotingPlatform/ConfigureCommand.php

```diff
@@ -446,6 +487,84 @@ private function configureNewElectionForTerritorialCouncil(
         $this->dispatcher->dispatch(new VotingPlatformElectionVoteIsOpenEvent($election));
     }
 
+    private function configureNewElectionForCommitteeSupervisor(Election $election): void
+    {
+        $designation = $election->getDesignation();
+
+        if (!$designation->isCommitteeSupervisorType()) {
+            return;
+        }
+
+        $committee = $election->getElectionEntity()->getCommittee();
+
+        $memberships = $this->entityManager->getRepository(CommitteeMembership::class)->findVotingForElectionMemberships($committee, $designation);
+
+        $list = $this->createVoterList(
+            $election,
+            array_map(function (CommitteeMembership $membership) { return $membership->getAdherent(); }, $memberships)
+        );
+
+        // Mark as Ghost voter adherent who can vote in many committees
+        foreach ($list->getVoters() as $voter) {
+            if (\count($voter->getVotersListsForDesignation($designation)) > 1) {
+                $voter->setIsGhost(true);
+            }
+        }
+
+        $this->entityManager->persist($list);
+        $this->entityManager->persist($election);
+        $this->entityManager->flush();
+    }
+
+    private function configureCandidatesGroupsForCommitteeSupervisorElection(Election $election): void
+    {
+        $designation = $election->getDesignation();
+
+        if (!$designation->isCommitteeSupervisorType()) {
+            return;
+        }
+
+        $electionRound = $election->getCurrentRound();
+        $committee = $election->getElectionEntity()->getCommittee();
+
+        // Create candidates groups
+        $candidacies = $this->entityManager->getRepository(CommitteeCandidacy::class)->findConfirmedByCommittee($committee, $designation);
+
+        $pools = [
+            $pool = new ElectionPool(ElectionPoolCodeEnum::COMMITTEE_SUPERVISOR),
+        ];
+
+        foreach ($candidacies as $candidacy) {
```

Comment:
could you use `committeeElection->getCandidaciesGroups()` here and loop on group (list) first instead of one collection of candidacies